{u'description': [u'Gentoo cross-compilation overlay for embedded devices'],
 u'feeds': [u'https://cgit.gentoo.org/proj/embedded-cross.git/atom/'],
 u'homepage': u'https://cgit.gentoo.org/proj/embedded-cross.git/',
 u'irc': None,
 u'name': u'embedded-cross',
 u'official': True,
 u'owner_email': u'armin76@gentoo.org',
 u'owner_name': u'Ra\xfal Porcel',
 u'priority': 50,
 u'quality': u'experimental',
 u'sources': [(u'git://anongit.gentoo.org/proj/embedded-cross.git',
               u'Git',
               u''),
              (u'https://anongit.gentoo.org/git/proj/embedded-cross.git',
               u'Git',
               u''),
              (u'git+ssh://git@git.gentoo.org/proj/embedded-cross.git',
               u'Git',
               u'')],
 u'status': u'official',
 u'supported': True}
$ pmaint sync /home/mgorny/repos/embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at location '/home/mgorny/repos/eigenlay' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/eigenlay', named '<unlabeled repository /home/mgorny/repos/eigenlay>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
Already up-to-date.
*** syncing '/home/mgorny/repos/embedded-cross'...
*** synced '/home/mgorny/repos/embedded-cross'
 * Sync succeeded
$ pmaint regen -t 2 embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'

!!! ERROR: app-misc/mdbus-9999 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: x11-terms/vala-terminal-1.2 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:

!!! ERROR: x11-terms/vala-terminal-1.1.1 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: dev-libs/libnl-2.0_pre9999 failed.
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '165' 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '218' 'depend'
 * Use PYTHON_DEPEND variable instead of NEED_PYTHON variable.

!!! ERROR: dev-python/gst-python-0.10.16 failed.
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 

!!! ERROR: media-plugins/gst-plugins-ivorbis-0.10.9 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '208' 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'

!!! ERROR: dev-libs/libgee-0.5.0.1 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:

!!! ERROR: dev-python/pyrtc-20090418 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '204' 'depend'
!!!                  ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!                  ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: sys-devel/gcc-4.4.3 failed.
!!!       vala-terminal-1.2.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: x11-drivers/xf86-video-glamo-20100626 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                  ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '261' 'depend'

!!! ERROR: sys-devel/gcc-4.4.2 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-libs/libnl/libnl-2.0_pre9999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '215' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-libs/libnl/libnl-2.0_pre9999.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '189' 'depend'
!!!                  ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'

!!! ERROR: dev-python/pyrtc-9999 failed.
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                 ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              mdbus-9999.ebuild, line   5:  Called inherit 'git'
!!!                 ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-libs/libnl/libnl-2.0_pre9999.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '189' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                          ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/media-plugins/gst-plugins-ivorbis/gst-plugins-ivorbis-0.10.9.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!       libnl-2.0_pre9999.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!                          ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/media-plugins/gst-plugins-ivorbis/gst-plugins-ivorbis-0.10.9.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: dev-python/pygobject-2.18.0 failed.
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/python-e_dbus/python-e_dbus-20090614.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/python-edje/python-edje-20090614.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:

!!! ERROR: x11-drivers/xf86-video-glamo-9999 failed.
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/gst-python/gst-python-0.10.16.ebuild'
!!!                 ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '248' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild'
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/media-plugins/gst-plugins-ivorbis/gst-plugins-ivorbis-0.10.9.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!     vala-terminal-1.1.1.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '166' 'depend'

!!! ERROR: dev-python/python-elementary-20090614 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/gst-python/gst-python-0.10.16.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                  ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-libs/libgee/libgee-0.5.0.1.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '166' 'depend'
!!!   gst-plugins-ivorbis-0.10.9.ebuild, line   5:  Called inherit 'gst-plugins-bad'
!!!                 ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '203' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gst-plugins-bad'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/gst-python/gst-python-0.10.16.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/pygobject/pygobject-2.18.0.ebuild'

!!! ERROR: dev-python/python-efl_utils-20090614 failed.
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                  ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                         ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.3.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-libs/libgee/libgee-0.5.0.1.ebuild'
!!!                   ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gst-plugins-bad.eclass'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/pygobject/pygobject-2.18.0.ebuild'
!!!                  ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '251' 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.2.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '169' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.3.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                  ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                         ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!      gst-python-0.10.16.ebuild, line   7:  Called inherit 'autotools' 'eutils' 'multilib' 'python'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/pygobject/pygobject-2.18.0.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gst-plugins-bad.eclass'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.2.ebuild'
!!!                          ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-elementary/python-elementary-20090614.ebuild'
!!!          pyrtc-20090418.ebuild, line   5:  Called inherit 'distutils' 'git'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.3.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                 ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-libs/libgee/libgee-0.5.0.1.ebuild'
!!!                          ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-elementary/python-elementary-20090614.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'python'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.2.ebuild'
!!!                 ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '228' 'depend'
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-elementary/python-elementary-20090614.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/python.eclass'
!!!   xf86-video-glamo-20100626.ebuild, line  16:  Called inherit 'x-modular'
!!!                 ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '246' 'depend'
!!!              gst-plugins-bad.eclass, line  19:  Called inherit 'eutils' 'gst-plugins10'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!        pygobject-2.18.0.ebuild, line   5:  Called inherit 'alternatives' 'autotools' 'gnome2' 'python' 'virtualx'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/python.eclass'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-9999.ebuild'
!!!                  python.eclass, line 277:  Called die
!!! The die message:
!!!  NEED_PYTHON variable is banned


!!! ERROR: dev-python/python-epsilon-20090614 failed.
!!!   python-elementary-20090614.ebuild, line   7:  Called inherit 'enlightenment_svn-python'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gst-plugins10'
!!!                 ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!               gcc-4.4.2.ebuild, line  24:  Called inherit 'toolchain'
!!!                   ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gst-plugins10.eclass'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!                         ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-efl_utils/python-efl_utils-20090614.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!          libgee-0.5.0.1.ebuild, line   5:  Called inherit 'gnome2'
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gst-plugins10.eclass'
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!               gcc-4.4.3.ebuild, line  24:  Called inherit 'toolchain'
!!!                gst-plugins10.eclass, line  37:  Called die
!!! The die message:
!!!  EAPI="0" is not supported anymore

!!!                         ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-efl_utils/python-efl_utils-20090614.ebuild'
!!!               ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-9999.ebuild'

!!! ERROR: x11-base/xglamo-20081205-r2 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'alternatives'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'toolchain'
!!!             isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-efl_utils/python-efl_utils-20090614.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-9999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gnome2'
!!!                   ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 176' for eclass enlightenment_svn-python

received unknown com: export CATEGORY='x11-libs' PR=r0 PVR='2.14.5' PV='2.14.5' EBUILD='/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild' P='gtk+-2.14.5' PF='gtk+-2.14.5' PN='gtk+'path
!!!               ebuild-daemon.bash, line 209:  Called __ebd_main_loop
Exception in thread Thread-7:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              pyrtc-9999.ebuild, line   5:  Called inherit 'distutils' 'git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'toolchain'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!!               ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '236' 'depend'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 201' for eclass alternatives

received unknown com: export CATEGORY='x11-libs' PR=r0 PVR='0.5.0.063' PV='0.5.0.063' EBUILD='/home/mgorny/repos/embedded-cross/x11-libs/e_dbus/e_dbus-0.5.0.063.ebuild' P='e_dbus-0.5.0.063' PF='e_dbus-0.5.0.063' PN='e_dbus'path
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!   python-efl_utils-20090614.ebuild, line   5:  Called inherit 'enlightenment_svn-python'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 241' for eclass distutils

!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
received unknown com: export CATEGORY='net-libs' PR=r0 PVR='0.1_p20090703' PV='0.1_p20090703' EBUILD='/home/mgorny/repos/embedded-cross/net-libs/webkit-efl/webkit-efl-0.1_p20090703.ebuild' P='webkit-efl-0.1_p20090703' PF='webkit-efl-0.1_p20090703' PN='webkit-efl'path
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                  ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!!               toolchain.eclass, line  29:  Called die
!!! The die message:
!!!  Need to upgrade to at least EAPI=2

!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '201' 'depend'
!!!               toolchain.eclass, line  29:  Called die
!!! The die message:
!!!  Need to upgrade to at least EAPI=2

!!!               ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/python-etk/python-etk-20090614.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!!                       ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20090614.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                  gnome2.eclass, line  19:  Called die
!!! The die message:
!!!  EAPI= is not supported


!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5.5 failed.
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!!             isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                  ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 179' for eclass enlightenment_svn-python

received unknown com: export CATEGORY='dev-lang' PR=r0 PVR='0.8.1.1' PV='0.8.1.1' EBUILD='/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild' P='vala-0.8.1.1' PF='vala-0.8.1.1' PN=valapath
!!!   xf86-video-glamo-9999.ebuild, line  10:  Called inherit 'x-modular'
Exception in thread Thread-9:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!!                      ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                       ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20090614.ebuild'
!!!                   x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 

!!! ERROR: sys-mobilephone/fsodatad-0.1.0 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: app-misc/mdbus-9999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.1.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: dev-libs/libnl-2.0_pre9999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: dev-python/pyrtc-20090418
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-20100626
!!! caught exception Metadata Exception: pkg ebuild src: sci-geosciences/fso-gpsd-0.8, attr eapi
error: parsed eapi doesn't match sourced eapi while processing ebuild src: sci-geosciences/fso-gpsd-0.8
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/pyrtc-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/pygobject-2.18.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-elementary-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-efl_utils-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-epsilon-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/gst-python-0.10.16
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: media-plugins/gst-plugins-ivorbis-0.10.9
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-base/xglamo-20081205-r2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/gtk+-2.14.5
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-20090704
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/e_dbus-0.5.0.063
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/webkit-efl-0.1_p20090703
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/fso-alsa-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-devel/gcc-4.4.3
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-devel/gcc-4.4.2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-libs/libgee-0.5.0.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/vala-0.8.1.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fsodatad-0.1.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while proc
!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5.1 failed.
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                  ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'

!!! ERROR: sys-mobilephone/fso-specs-2010.05.11.2 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!                      ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:

!!! ERROR: dev-python/python-epsilon-20091202 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!           isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20090614.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!      xglamo-20081205-r2.ebuild, line   5:  Called inherit 'autotools' 'git' 'toolchain-funcs'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                      ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '285' 'depend'
!!!              ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '203' 'depend'
!!!                      ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   python-epsilon-20090614.ebuild, line   5:  Called inherit 'enlightenment_svn-python'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!              ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '245' 'depend'
!!!               ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'

!!! ERROR: sys-mobilephone/fsogsmd-0.5.0 failed.

!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!!     ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!! Call stack:
!!!                ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 183' for eclass enlightenment_svn-python

!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='2.0.0' PV='2.0.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild' P='apm2-2.0.0' PF='apm2-2.0.0' PN=apm2path
Exception in thread Thread-13:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!                      ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                    ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!               ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                              ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                      ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!               x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!                      ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '285' 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                    ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                              ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!              ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!          fsodatad-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                      ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                    ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '275' 'depend'
!!!               ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '236' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                              ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'

!!! ERROR: x11-libs/elementary-0.6.0.063 failed.
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!                              ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                    ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                  isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sys-mobilephone/fsonetworkd-0.1.0 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!!   freesmartphone-framework-0.5.1.ebuild, line   6:  Called inherit 'git'
!!! Call stack:
!!!               ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                  isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '198' 'depend'
!!!            ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                       ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20091202.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                       ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20091202.ebuild'
!!!                      ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
!!!                            ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!!           isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20091202.ebuild'
!!!   freesmartphone-framework-0.5.5.ebuild, line   6:  Called inherit 'git'
!!!                       ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!   python-epsilon-20091202.ebuild, line   5:  Called inherit 'enlightenment_svn-python'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sys-mobilephone/fsotdld-0.3.0 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!            ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                       ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '219' 'depend'
!!!     ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!!                ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 208' for eclass enlightenment_svn-python

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.4' PV='0.9.4' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsousaged/fsousaged-0.9.4.ebuild' P='fsousaged-0.9.4' PF='fsousaged-0.9.4' PN=fsousagedpath
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '218' 'depend'
!!!                            ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'

!!! ERROR: sys-mobilephone/fsodeviced-0.9.4 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sys-mobilephone/libframeworkd-phonegui-efl-20090724 failed.
!!!           fsogsmd-0.5.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                      ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '198' 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-libs/elementary/elementary-0.6.0.063.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '213' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-libs/elementary/elementary-0.6.0.063.ebuild'
!!!                           ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-libs/elementary/elementary-0.6.0.063.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!   freesmartphone-framework-0.5.ebuild, line   5:  Called inherit 'distutils'
!!!          isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
!!!                           ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!          ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!    elementary-0.6.0.063.ebuild, line   6:  Called inherit 'enlightenment'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 266' for eclass distutils

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090614 PV=20090614 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild' P='libframeworkd-glib-20090614' PF='libframeworkd-glib-20090614' PN='libframeworkd-glib'path
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe


!!! ERROR: sys-mobilephone/fsoinitd-0.1.0 failed.
!!!                           ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '306' 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!   fso-specs-2010.05.11.2.ebuild, line   6:  Called inherit 'base' 'autotools' 'git'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!!        fsodeviced-0.9.4.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 306' for eclass enlightenment

!!!       fsonetworkd-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090614 PV=20090614 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090614.ebuild' P='libframeworkd-phonegui-efl-20090614' PF='libframeworkd-phonegui-efl-20090614' PN='libframeworkd-phonegui-efl'path
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
Exception in thread Thread-15:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                           ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090506 PV=20090506 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090506.ebuild' P='libframeworkd-phonegui-20090506' PF='libframeworkd-phonegui-20090506' PN='libframeworkd-phonegui'path
!!!           fsotdld-0.3.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass cornucopia_git

Exception in thread Thread-8:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090724 PV=20090724 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild' P='libframeworkd-phonegui-20090724' PF='libframeworkd-phonegui-20090724' PN='libframeworkd-phonegui'path
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
Exception in thread Thread-14:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                                   ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090724.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!               ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sys-mobilephone/libframeworkd-glib-20090227 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                   ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sys-mobilephone/libframeworkd-glib-0.1 failed.
!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                                   ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090724.ebuild'
!!!                   ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '266' 'depend'
!!!              ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090614 PV=20090614 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090614.ebuild' P='libframeworkd-phonegui-20090614' PF='libframeworkd-phonegui-20090614' PN='libframeworkd-phonegui'path
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '203' 'depend'
essing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-epsilon-20091202
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fso-specs-2010.05.11.2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fsogsmd-0.5.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/apm2-2.0.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-glib-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/elementary-0.6.0.063
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsonetworkd-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsotdld-0.3.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsodeviced-0.9.4
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsousaged-0.9.4
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-99999999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090724
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090614
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libfsoresource-0.1.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fsoinitd-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090614
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libgsm0710mux-0.9.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090506
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libphone-ui-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090724
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libphone-utils-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090227
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/mioctl-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-0.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/mterm2-1.9.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090614
!!! cauException in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                       isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090724.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!   libframeworkd-phonegui-efl-20090724.ebuild, line   5:  Called inherit 'shr'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!                 ebuild-default-functions.lib, line 230:  Called __internal_inherit 'shr'
!!!                            ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/shr.eclass'
!!!                   ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                           ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!              ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '245' 'depend'
!!!                           ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!               isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!                       isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/eclass/shr.eclass'
!!!   libframeworkd-glib-20090227.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!                                   shr.eclass, line   5:  Called inherit 'autotools' 'eutils' 'git'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                 ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!          fsoinitd-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'

!!! ERROR: sys-mobilephone/libfsoframework-0.8.1 failed.
!!!                            ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed
!!!              ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'

!!!                      ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'

!!! ERROR: sys-mobilephone/libfsobasics-0.9.9 failed.
!!!                      ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 193' for eclass git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.0' PV='0.1.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/mioctl/mioctl-0.1.0.ebuild' P='mioctl-0.1.0' PF='mioctl-0.1.0' PN=mioctlfailed
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!          isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '238' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   libframeworkd-glib-0.1.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '223' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'

!!! ERROR: sys-mobilephone/libgsm0710-1.2.1 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!!               ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 193' for eclass git
!!! Call stack:

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='1.9.0' PV='1.9.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/mterm2/mterm2-1.9.0.ebuild' P='mterm2-1.9.0' PF='mterm2-1.9.0' PN=mterm2failed
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsobasics/libfsobasics-0.9.9.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
Exception in thread Thread-11:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsoframework/libfsoframework-0.8.1.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsoframework/libfsoframework-0.8.1.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsoframework/libfsoframework-0.8.1.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '213' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsobasics/libfsobasics-0.9.9.ebuild'
!!!   libfsoframework-0.8.1.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsobasics/libfsobasics-0.9.9.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libgsm0710/libgsm0710-1.2.1.ebuild'
!!!      libfsobasics-0.9.9.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 219' for eclass cornucopia_git

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libgsm0710/libgsm0710-1.2.1.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libgsm0710/libgsm0710-1.2.1.ebuild'
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/phonefsod/phonefsod-99999999.ebuild' P='phonefsod-99999999' PF='phonefsod-99999999' PN=phonefsodpath
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!        libgsm0710-1.2.1.ebuild, line   6:  Called inherit 'base'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 224' for eclass base

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090901 PV=20090901 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/pyphonelog/pyphonelog-20090901.ebuild' P='pyphonelog-20090901' PF='pyphonelog-20090901' PN=pyphonelogpath
Exception in thread Thread-16:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sys-mobilephone/ophonekitd-99999999 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '224' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/ophonekitd/ophonekitd-99999999.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/ophonekitd/ophonekitd-99999999.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/ophonekitd/ophonekitd-99999999.ebuild'
!!!     ophonekitd-99999999.ebuild, line  22:  Called inherit 'shr'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'shr'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 236' for eclass shr

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/shr-contacts/shr-contacts-99999999.ebuild' P='shr-contacts-99999999' PF='shr-contacts-99999999' PN='shr-contacts'path
Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

ght exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/ophonekitd-20090614
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libfsoframework-0.8.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfsobasics-0.9.9
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/mkdump-1.0.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phoneui-apps-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libgsm0710-1.2.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/ophonekitd-20090506
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/pyphonelog-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/phonefsod-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/pyphonelog-20090901
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/shr-contacts-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/ophonekitd-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/shr-contacts-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/shr-dialer-20090506
Error running an operation:
 Unhandled exception in regen_cache
 * Cache regen failed with 246
